### DISCUSSION TITLE
Multi-Colored Filaments

### DISCUSSION THREAD

**Discussion by thewolfman56** [Disane87/spoolman-homeassistant#148] [Q&A] [answered]
Whenever I add a multi-colored filament to Spoolman, the new spool shows up with just the nozzle icon and the theme color.  Is there something I need to do so that colors show up similar to the single-colored filaments?  Multi-colored filaments look just fine in Spoolman itself.

**Comment by Disane87** [comment]
At the moment I never tested multi color filament because I don't have one. Can you provide me a screenshot how it looks in spoolman? Would be nice so I can evaluate a way to support this in a later release.

**Comment by thewolfman56** [comment]
Of course.  Please let me know if there is anything else that I can provide that could help.

![image](https://github.com/user-attachments/assets/28404377-2afd-4a85-a5ec-5132bbcffbf6)

![image](https://github.com/user-attachments/assets/e165ba0e-0e30-48cc-8375-e2dd0fc0e8c7)

**Comment by Disane87** [comment]
Should be implemented so I close this discussion

**Reply by thewolfman56** [reply, marked as answer]
It is implemented.  My apologies.  Looks good.
